CMakeLists.txt

```diff
@@ -24,61 +27,20 @@ endif ()
 
 find_package(MPI REQUIRED)
 
-# Add include paths
-if (FIBER_ENABLE_HEFFTE)
-add_definitions(-DFIBER_ENABLE_HEFFTE)
-include_directories($ENV{HOME}/heffte/build/include)
-link_directories($ENV{HOME}/heffte/build)
-endif()
-
-if (FIBER_ENABLE_FFTMPI)
-add_definitions(-DFIBER_ENABLE_FFTMPI)
-include_directories($ENV{HOME}/Benchmarks_FFT/fftmpi/src)
-link_directories($ENV{HOME}/Benchmarks_FFT/fftmpi/src)
-endif()
-
-if (FIBER_ENABLE_ACCFFT)
-add_definitions(-DFIBER_ENABLE_ACCFFT)
-include_directories($ENV{HOME}/Benchmarks_FFT/accfft/build/include)
-link_directories($ENV{HOME}/Benchmarks_FFT/accfft/build)
-endif()
-
-if (FIBER_ENABLE_P3DFFT)
-add_definitions(-DFIBER_ENABLE_P3DFFT)
-include_directories($ENV{HOME}/Benchmarks_FFT/p3dfft/include)
-link_directories($ENV{HOME}/Benchmarks_FFT/p3dfft/build)
-endif()
+# Location of the FFT library
+include_directories(${FIBER_FFT_INCLUDE_DIR})
+link_directories(${FIBER_FFT_LIB_DIR})
 
-if (FIBER_ENABLE_FFTE)
-add_definitions(-DFIBER_ENABLE_FFTE)
-include_directories($ENV{HOME}/Benchmarks_FFT/ffte-7.0/include)
-link_directories($ENV{HOME}/Benchmarks_FFT/ffte-7.0/src)
-endif()
-
-if (FIBER_ENABLE_SWFFT)
-add_definitions(-DFIBER_ENABLE_SWFFT)
```

Comment:
`add_definitions()` adds a compile-time flag. Now the flag is missing. Is this intended?